src/wp-includes/script-loader.php

```diff
@@ -827,7 +827,7 @@ function wp_default_scripts( $scripts ) {
 	$scripts->add( 'jquery-color', '/wp-includes/js/jquery/jquery.color.min.js', array( 'jquery' ), '2.1.2', 1 );
 	$scripts->add( 'schedule', '/wp-includes/js/jquery/jquery.schedule.js', array( 'jquery' ), '20m', 1 );
 	$scripts->add( 'jquery-query', '/wp-includes/js/jquery/jquery.query.js', array( 'jquery' ), '2.1.7', 1 );
-	$scripts->add( 'jquery-serialize-object', '/wp-includes/js/jquery/jquery.serialize-object.js', array( 'jquery' ), '0.2', 1 );
+	$scripts->add( 'jquery-serialize-object', '/wp-includes/js/jquery/jquery.serialize-object.js', array( 'jquery' ), '0.2-wp', 1 );
 	$scripts->add( 'jquery-hotkeys', "/wp-includes/js/jquery/jquery.hotkeys$suffix.js", array( 'jquery' ), '0.0.2m', 1 );
```

Comment:
This will need a version change too.

Reply:
I'm not sure if it does (this is referring to `jquery-hotkeys`, yes?), it's already been modified by core, and the core version string has been suffixed with `m` (which appears to have been used in the past instead of the `-wp` used now).

Core has made manual edits to this as far back as 2008